app/renderer/playerView.js

```diff
@@ -107,41 +107,59 @@ function addTextSection(text)
     // Split the line if we have a >>
     var splitOnCarrotTags = text.split(">>");
     var nonTagText = splitOnCarrotTags.shift();
+
     // Reconstitute the rest of the line that has tags
     var tagText = "";
     if (splitOnCarrotTags.length > 0) {
         tagText = ">>" + splitOnCarrotTags.join(">>");
     }
 
     // If all we have left is a character name, style that like a tag
+
     if (/[A-Z]+:\s*$/.test(nonTagText)) {
         tagText = nonTagText + tagText;
         nonTagText = "";
     }
 
-    var splitIntoSpans = nonTagText.split(" ");
+    var arrayOfWords = nonTagText.split(" ");
+    var dialogueRegex = /^[A-Z]+:/
```

Comment:
Worth noting that, while more restrictive than the characterNamePattern Regex we use in C#, all of our current Ink files are correctly matched by this one... So there's something to be said for sticking with something that's simpler and works. I took a look at the C# version and I think the main difference is that it requires that only the very first character to be an uppercase alphanumeric... thereafter a single space or lowercase letter are allowed. The original idea was to allow for names like `McNASTY:` and `JULIUS CAESAR` but so far we don't have any names like this, and they're no longer displayed onscreen, making the point rather moot IMO.